cmd/neofs-node/netmap.go

```diff
@@ -22,6 +22,11 @@ import (
 	"go.uber.org/zap"
 )
 
+// defaultEpochDuration sets default epoch duration. Should not be used in a
+// normal situations but if no info can be fetched, smth should be used as a
```

Comment:
`if no info can be fetched, smth should be used` sounds like a philosophy to me. Simply `defaultEpochDuration is a default epoch duration to replace zero from FS chain.`

Reply:
not a philosophy at all to me. my version explains when it is used and states that zero value is not ok, your version repeats `if` statement when this const is used. fixed